src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -475,6 +475,56 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 TextSpan,
                 "BCP087",
                 "Array and object literals are not allowed here.");
+
+            public ErrorDiagnostic UnableToFindPathForModule() => new ErrorDiagnostic(
+                TextSpan,
+                "BCP088",
+                "Unable to find file path for module.");
+
+            public ErrorDiagnostic ErrorOccurredLoadingModule(string failureMessage) => new ErrorDiagnostic(
+                TextSpan,
+                "BCP089",
+                $"An error occurred loading the module. Received failure \"{failureMessage}\".");

```

Comment:
I haven't looked at the code before looking at the errors, but I'm wondering if we can do one of the following:
1. Change to `$"An error occurred loading the module. {failureMessage}"` (the received failure thing seems like extra noise that isn't helpful assuming the failure message is a complete sentence)
2. Have a specific error for all expected failure cases.

Reply:
I'll go with 1. I wasn't sure I could rely on the formatting of the exception, which is why I quoted it; let's change it and see what happens!

Reply:
Yeah if we're passing in an IOException.Message here, they will be complete sentences. When we have loc, there might be a mismatch between languages if dotnet language set is different than our language set, but that's a future us problem 🙂.